teams/team_plumed.md

```diff
@@ -0,0 +1,18 @@
+# Team Plumed
+
+## Contributors 
+* Andy Hillhouse
+    * netid: ahillhou
+    * Github username: ahillhou
+* Bryson Gullett
+    * netid: bgullet1
+    * github username: BGull00
+* Tristan Ainley
+    * netid: tainley
+    * github username: tristan-ainley
+* Juliet Bradford
+    * netid: jbradf19
+    * github username: jAllieB
+

```

Comment:
@jmuncy2 should be listed

Reply:
I have added him and (at least attempted to) push teams/team_plumed.md

Reply:
I pushed it in #166